action.yml

```diff
@@ -4,17 +4,21 @@ inputs:
   GITHUB_TOKEN:
     description: "GitHub token to interact with the repository."
     required: true
-  OPENAI_API_KEY:
-    description: "OpenAI API key for GPT."
+  INFERENCE_API_KEY:
```

Comment:
Consider providing more context in the description of `INFERENCE_API_KEY` to clarify its usage and any specific requirements for the key.